### PR TITLE
grant page and lil adjustment on the index page

### PR DESCRIPTION
**Pull request by opeyemipeter1759**

### CODE REVIEW

File: grant.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Home | SME-Grants</title>
```
Comment (JoyChukwukere) [inline grant.html:12]: change your document title to the title of the grant | SME-Grants

File: styles.css [hunk 2]
```diff
@@ -80,34 +99,82 @@ Forms Page CSS Begins
...
+.btn:hover {
+    color: grey;
+}
+
+.bg-purple {
+    background-color: #14A2B8 !important;
+    color: #FFF;
+}
+
+.bg-red {
```
Comment (JoyChukwukere) [inline styles.css:173]: Are we using red colours now?

**Review by JoyChukwukere — APPROVED** [review]

File: grant.html (hunk 1 above)
Comment (opeyemipeter1759) [reply grant.html:12]: okay,

File: styles.css (hunk 2 above)
Comment (opeyemipeter1759) [reply styles.css:173]: i used it for the card, when you merge and pull you will see it

**Pull request merged**